advanced_filters/forms.py

```diff
@@ -230,7 +230,7 @@ def forms(self):
 
 AFQFormSet = formset_factory(
     AdvancedFilterQueryForm, formset=AdvancedFilterFormSet,
-    extra=1, can_delete=True)
+    extra=0, can_delete=True)
 
 AFQFormSetNoExtra = formset_factory(
```

Comment:
AFQFormSetNoExtra, defined [just below (line 235)](https://github.com/worthwhile/django-advanced-filters/blob/735debaccee7205032fe35686a4c453f2f081f2e/advanced_filters/forms.py#L235), already has 0 extra forms. And I believe the current default is used when creating a new filter, which would make sense wince a filter with 0 rules is not really interesting.

Perhaps it would be better to add an require an extra form only when adding, but not when editing rules? We can [toggle between the forms](https://github.com/worthwhile/django-advanced-filters/blob/735debaccee7205032fe35686a4c453f2f081f2e/advanced_filters/forms.py#L364) by [passing `extra_form=True/False` in `admin.py`](https://github.com/worthwhile/django-advanced-filters/blob/735debaccee7205032fe35686a4c453f2f081f2e/advanced_filters/forms.py#L364).